### PR TITLE
Fix pymedusa/SickRage/issues/542

### PR DESCRIPTION
**Pull request by fernandog**
As discussed and suggested by @OmgImAlexis 

https://github.com/pymedusa/SickRage/issues/542

### CODE REVIEW

**Comment by fernandog** [comment]
@p0psicles can you test please?

File: gui/slick/views/config_providers.mako [hunk 1]
```diff
@@ -435,7 +435,7 @@ $('#config-components').tabs();
                             <label for="${curTorrentProvider.get_id()}_ratio">
                                 <span class="component-title" id="${curTorrentProvider.get_id()}_ratio_desc">Seed ratio:</span>
                                 <span class="component-desc">
-                                    <input type="number" min="-1" step="0.1" name="${curTorrentProvider.get_id()}_ratio" id="${curTorrentProvider.get_id()}_ratio" value="${curTorrentProvider.ratio}" class="form-control input-sm input75" />
+                                    <input type="number" min="-1" step="0.1" name="${curTorrentProvider.get_id()}_ratio" id="${curTorrentProvider.get_id()}_ratio" value="${"" if curTorrentProvider.ratio is None else curTorrentProvider.ratio}" class="form-control input-sm input75" />
```
Comment (p0psicles) [inline gui/slick/views/config_providers.mako]: Change this: value="${"" if
Into:
value="${'' if

You need to use single quotes. If you change that, it's okay for me. No more errors.

**Comment by fernandog** [comment]
Approved

**Comment by p0psicles** [comment]
Approved

**Comment by p0psicles** [comment]
https://github.com/pymedusa/SickRage/issues/392

**Pull request merged**